src/lib/router.ts

```diff
@@ -30,4 +32,69 @@ router.post('/matches', async (req, res, next) => {
   }
 });
 
+type Highlight = {
+  timestamp: number;
+  type: string;
+  videoSrc: string;
+};
```

Comment:
I would move this to the other `type` in `matches.ts`. 
Maybe call it `MatchHighlight`, because it's a part of a match (up to you).

Reply:
yes right, would be more obvious to find it there